src/BigT/rowJoin.java

```diff
@@ -0,0 +1,271 @@
+//package BigT;
+//import global.AttrOperator;
+//import global.AttrType;
```

Comment:
The whole class is commented out. May be you want to rename the other one as RowJoin and delete this.